compiler/rustc_trait_selection/src/traits/error_reporting/type_err_ctxt_ext.rs

```diff
@@ -498,10 +498,6 @@ impl<'tcx> TypeErrCtxt<'_, 'tcx> {
                                 )
                             })
                             .unwrap_or_default();
-                        let file_note = file.as_ref().map(|file| format!(
-                            "the full trait has been written to '{}'",
-                            file.display(),
-                        ));
```

Comment:
Hmm, do we lose a diagnostic for where the trait has been written to if we remove this?

Reply:
aaaah I'll double check.
It looks like we don't have any tests here actually.